scripts/prepare_sector_network.py

```diff
@@ -2477,7 +2477,6 @@ def add_biomass(n, costs):
         )
 
     if biomass_potentials.filter(like="unsustainable").sum().sum() > 0:
-        add_carrier_buses(n, "oil")
```

Comment:
intended?

Reply:
Yes, was redundant to the addition in line 2539.